frontend/src/locales/en.json

```
@@ -503,6 +503,7 @@
       }
     },
     "campCreate": {
+      "couponKeyHint": "For the closed beta a coupon code is necessary. On ecamp3.ch/closed-beta you can find out how to get such an coupon code.",
```

```suggestion
      "couponKeyHint": "For the closed beta, a coupon code is required. On ecamp3.ch/closed-beta, you can find out how to get such an coupon code.",
```
"Necessary" would imply that the user feels like this is needed. "Required" implies that the developers say it's needed.